.cirrus.yml

```diff
@@ -0,0 +1,40 @@
+task:
+  env:
+    HOME: /tmp  # cargo cache needs it
+    VERSION: nightly
+  matrix:
+    - name: FreeBSD 12 amd64 nightly   
+      freebsd_instance:
+        image: freebsd-12-3-release-amd64
+    - name: FreeBSD 13 amd64 nightly
+      freebsd_instance:
+        image: freebsd-13-1-release-amd64
+    - name: FreeBSD 14 amd64 nightly
+      freebsd_instance:
+        image: freebsd-14-0-snap
```

Comment:
I think `freebsd-14-0-snap` is an image family. Running the following does yield lots of possible images:

```
gcloud compute images list --project freebsd-org-cloud-dev --no-standard-images | rg freebsd-14
```

`freebsd-14-0-current-amd64-v20221110` seems to work here.

Reply:
Ahh, I see.  We would have to do `image_family: freebsd-14-0-snap` in order to use the snapshot.